### PR TITLE
Cellpose2.0.5 container

### PR DESCRIPTION
**Pull request by sunyi000**
docker container for cellpose v2.0.5
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER

### CODE REVIEW

**Comment by sunyi000** [comment]
@osallou could you please have a look, thank you very much

File: cellpose/2.0.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+FROM python:3.8-slim
+
+LABEL base_image="python:2.8-slim"
+LABEL version="0"
```
Comment (osallou) [inline cellpose/2.0.5/Dockerfile]: should be version 1

File: cellpose/2.0.5/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,56 @@
+FROM python:3.8-slim
+
+LABEL base_image="python:2.8-slim"
```
Comment (osallou) [inline cellpose/2.0.5/Dockerfile]: is 3.8-slim ;-)

**Review by osallou — CHANGES_REQUESTED** [review]

File: cellpose/2.0.5/Dockerfile (hunk 2 above)
Comment (sunyi000) [reply cellpose/2.0.5/Dockerfile]: changed. thank you!

**Review by osallou — APPROVED** [review]

**Pull request merged**